buildSrc/src/main/groovy/org/elasticsearch/gradle/test/ClusterFormationTasks.groovy

```diff
@@ -358,6 +375,34 @@ class ClusterFormationTasks {
         return copyPlugins
     }
 
+    /** Configures task to copy a plugin based on a zip file resolved using dependencies for an older version */
+    static Task configureCopyBwcPluginsTask(String name, Project project, Task setup, NodeInfo node) {
+        for (Map.Entry<String, Project> plugin : node.config.plugins.entrySet()) {
+            System.err.println(plugin.getKey() + ": " + plugin.getValue())
+            Project pluginProject = plugin.getValue()
+            if (pluginProject.plugins.hasPlugin(PluginBuildPlugin) == false) {
+                throw new GradleException("Task ${name} cannot project ${pluginProject.path} which is not an esplugin")
+            }
+            String configurationName = "_plugin_bwc_${pluginProject.path}"
```

Comment:
I tried this. The issue I ran into is the code executed after this statement expects the configuration to have a single file. When using a single version, this is ok but when running BWC tests with multiple versions the configuration will result in having multiple files.